### PR TITLE
remove unused output that causes problems

### PR DESCRIPTION
**Pull request by fillup**
We found this output causes failed destroys due to the task definition no longer existing but trying to look it up for an output. It may also be the cause of problems we've seen during creations but I don't know for sure.

### CODE REVIEW

**Review by forevermatt — DISMISSED** [review]

**Review by Baggerone — DISMISSED** [review]

**Review by dalenewby — DISMISSED** [review]

File: aws/vpc-public-only/outputs.tf [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// Default Security Group ID
+output "id" {
```
Comment (forevermatt) [inline aws/vpc-public-only/outputs.tf:2]: This output isn't listed in the readme

File: aws/vpc-public-only/outputs.tf [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+output "public_subnet_cidr_blocks" {
+  value = ["${aws_subnet.public_subnet.*.cidr_block}"]
+}
+
+output "db_subnet_group_name" {
+  value = "${aws_db_subnet_group.db_subnet_group.name}"
+}
+
+output "aws_zones" {
```
Comment (forevermatt) [inline aws/vpc-public-only/outputs.tf:22]: This output also hasn't yet been added to the outputs section of the readme, if that matters.

File: aws/vpc-public-only/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+## Required Inputs
+
+ - `app_name` - Name of application, ex: Doorman, IdP, etc.
+ - `app_env` - Name of environment, ex: production, testing, etc.
+ - `aws_zones` - A list of zones to create subnets in, ex: `["us-east-1c", "us-east-1d", "us-east-1e"]`
+
+## Optional Inputs
+
+ - `enable_dns_hostnames` - default `false`
```
Comment (forevermatt) [inline aws/vpc-public-only/README.md:20]: The actual default value for this `enable_dns_hostnames` input (in the `vars.tf` file) is `true`, not `false`.

**Review by forevermatt — CHANGES_REQUESTED** [review]
Minor corrections to README needed (mainly the default value change).

File: aws/vpc-public-only/outputs.tf (hunk 1 above)
Comment (fillup) [reply aws/vpc-public-only/outputs.tf:2]: thanks

File: aws/vpc-public-only/outputs.tf (hunk 2 above)
Comment (fillup) [reply aws/vpc-public-only/outputs.tf:22]: thanks

File: aws/vpc-public-only/README.md (hunk 3 above)
Comment (fillup) [reply aws/vpc-public-only/README.md:20]: thanks

**Comment by fillup** [comment]
Closing PR to open a new one

**Pull request closed without merging**